index.json

```diff
@@ -1 +1 @@
nse": ["", "undecided"], "url": "https://storage.googleapis.com/models.cdn.sourced.tech/models%2Fid2vec%2F92609e70-f79c-46b5-8419-55726e873cfc.asdf", "references": [["Source code identifier embeddings", "https://blog.sourced.tech/post/id2vec/"]]}, "3467e9ca-ec11-444a-ba27-9fa55f5ee6c1": {"dependencies": [], "parent": "", "description": "A little under 1M     identifier embeddings, generated for identifiers extracted from half of PGA in June 2018. New pipeline was used, with splitting and stemming of identifiers, the full descriptio    n can be found in the \"Algorithms\" section of the [sourced.ml](https://github.com/src-d/ml) repository.", "version": [1, 0, 0], "extra": {"Size of each embedding": "300", "Number of tokens": "9    99,424", "Data collection date": "June 2018"}, "created_at": "2018-07-19 13:14:53.000621", "size": "1.2 GB", "code": "from sourced.ml.models import Id2Vec\nid2vec = Id2Vec().load(%s)\nprint(\"Number of tokens:\", len(id2vec))", "references": [["Source code identifier embeddings", "https://blog.sourced.tech/post/id2vec/"]], "url": "https://storage.googleapis.com/models.cdn.sourced.tech/models%2Fid2vec%2F3467e9ca-ec11-444a-ba27-9fa55f5ee6c1.asdf", "license": ["", "none"]}}}, "meta": {"typos_correction": {"code": "from lookout.style.typos.corrector import TyposCorrector\ncorrector = TyposCorrector().load(%s)\nprint(\"Corrector configuration:\\n\", corrector.dump())", "description": "Model that suggests fixes to correct typos.", "default": "d798e898-c6b2-4e39-809f-f502571584e8"}, "topics": {"code": "from sourced.ml.models import Topics\ntopics = Topics().load(%s)\nprint(\"Number of topics:\", len(topics))\nprint(\"Number of tokens:\", len(topics.tokens))", "description": "Topic modeling of Git repositories. All tokens are identifiers extracted from repositories and seen as indicators for topics. They are used to infer the topic(s) of repositories.", "default": "c70a7514-9257-4b33-b468-27a8588d4dfa"}, "bow": {"code": "from sourced.ml.models import BOW\nbow = BOW().load(%s)\nprint(\"Number of documents:\", len(bow))\nprint(\"Number of tokens:\", len(bow.tokens))", "description": "Weighted bag-of-words, that is, every bag is a feature extracted from source code and associated with a weight obtained by applying TFIDF.", "default": "1e3da42a-28b6-4b33-94a2-a5671f4102f4"}, "docfreq": {"code": "from sourced.ml.models import DocumentFrequencies\ndf = DocumentFrequencies().load(%s)\nprint(\"Number of tokens:\", len(df))", "description": "Document frequencies of features extracted from source code, that is, how many documents (repositories, files or functions) contain each tokenized feature.", "default": "f64bacd4-67fb-4c64-8382-399a8e7db52a"}, "id2vec": {"code": "from sourced.ml.models import Id2Vec\nid2vec = Id2Vec().load(%s)\nprint(\"Number of tokens:\", len(id2vec))", "description": "Source code identifier embeddings, that is, every identifier is represented by a dense vector.", "default": "92609e70-f79c-46b5-8419-55726e873cfc"}}}
```

Comment:
We should generate human-readable JSON. @r0mainK I cannot wait until you join us :)